app/utils/hentPathForSteg.ts

```diff
@@ -3,12 +3,12 @@ import { ESteg } from '~/typer/felles';
 export const hentPathForSteg = (steg: ESteg) => {
```

Comment:
Kjetil: Denne må endres senere ☺️

Reply:
Kort for dette ✨ (Prøver å unngå stor PR): 

https://trello.com/c/Jy9R9x4X/141-hentpathforsteg-m%C3%A5-refaktoreres-til-%C3%A5-tilpasse-st%C3%B8nad-typen